### PR TITLE
ProjectEuler1 ready to be reviewed.

### PR DESCRIPTION
**Pull request by csizmadiazsolt**
ProjectEuler1 added. Readme file still missing but will be added soon :)

### CODE REVIEW

File: ProjectEuler1/src/main/java/projecteulerone/SumOfMultiplies.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package projecteulerone;
+
+import java.util.List;
+
+public class SumOfMultiplies {
+	public int generateSumOfMultiplies(List<Integer> args) {
+		int sum = 0;
+
+		for (int i = 0; i < args.get(2); i++) {
```
Comment (ZsoltFabok) [inline ProjectEuler1/src/main/java/projecteulerone/SumOfMultiplies.java]: It is a good practice and it makes your code more readable if you extract the list entries to variables. For example, it is not clear at the first read what is `args.get(2)` in this context, not to mention `args.get(0)`

File: ProjectEuler1/src/test/java/projecteulerone/ProjectEuler1IntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+		assertEquals("", stdout.toString());
+		assertEquals("Incorrect parameter types. Needed types: <num> <num> <num>" + System.lineSeparator(),
+				stderr.toString());
+	}
+
+	@Test
+	public void prints_error_when_non_digital_arguments_and_too_much_arguments() {
+		ProjectEuler1.main("a", "b", "3", "4");
+
+		assertEquals("", stdout.toString());
```
Comment (ZsoltFabok) [inline ProjectEuler1/src/test/java/projecteulerone/ProjectEuler1IntegrationTest.java]: I have seen this before how about creating an `assertEmptyString()` method?

Comment (csizmadiazsolt) [inline ProjectEuler1/src/test/java/projecteulerone/ProjectEuler1IntegrationTest.java]: Fixed.

File: ProjectEuler1/src/main/java/projecteulerone/SumOfMultiplies.java (hunk 1 above)
Comment (csizmadiazsolt) [inline ProjectEuler1/src/main/java/projecteulerone/SumOfMultiplies.java]: Fixed.

**Pull request merged**